docs/getting-started/usage.md

```diff
@@ -6,107 +6,64 @@ order: 1
 
 ## Quick Start
 
-The following steps will create a starter app configured with Instructure-UI
-presets that is immediately ready for development. Recommended if you are starting from scratch.
+The following steps will create a React app that uses Instructure-UI. Recommended if you are starting from scratch.
 
-### Creating a Starter App
+1. Create a new react app, e.g. with [create-react-app](https://reactjs.org/docs/create-a-new-react-app.html):
 
-You can create the starter app using `npx` or by globally installing the `instui-cli`.
-
-#### Using npx
-
-```bash
-$ npx @instructure/instui-cli create app --name MyInstuiApp
-```
-
-#### Globally installing `instui-cli` with `yarn` or `npm`
-
-```bash
-$ yarn global add @instructure/instui-cli
-```
-
-or
-
-```bash
-$ npm install -g @instructure/instui-cli
+```shell
+npx create-react-app my-cool-app
 ```
 
-You can now create the app using the `instui` command
-
-```bash
-instui create app --name MyInstuiApp
-```
+This will generate a vanilla React app with tests, sample code and scripts. You can try it by running `yarn start`.
 
-### Start the Development Server
+2. Add InstUI dependencies to your `package.json`. We recommend that you add the `@instructure/ui` meta-package, this contains all of our UI components (substitute the version number with the latest one):
 
-```bash
-$ cd MyInstuiApp
+```json
+{
+  ...
+  "dependencies": {
+    ...
+    "@instructure/ui": "^8.6.0"
+  }
+}
 ```
 
-then
-
-```bash
-$ yarn
-$ yarn start
-```
+Run `yarn install`, so InstUI is downloaded to your `node_modules` folder and can be used.
 
-or
+3. Now you are ready to use InstUI, let's try it out. Replace the code in `App.js` with the following:
 
-```bash
-$ npm install
-$ npm start
-```
-
-Your app is running on [http://0.0.0.0:3000](http://0.0.0.0:3000). You can now
-edit `src` files and observe the changes. The app `README` contains more specific
-information on available commands and theming.
-
-## Integrating With an Existing Project
-
-Before using Instructure-UI components, you must first import a theme. All
-components are themeable, and themes control their look and feel. The component
-examples seen throughout the documentation use the [canvas theme](#canvas) by
-default which can be used by adding `@instructure/canvas-theme` to your project
-dependencies:
```

Comment:
This seems like a nice piece of information for people meeting InstUI for the first time, I would keep it.

Reply:
added it back